try:
    pass
except:
    pass
finally:
    match name_5:
        case 0:
            pass
name_2.name_3

# output:
# python: Python/flowgraph.c:511: no_redundant_jumps: Assertion `0' failed.
